xml/System.Text.Json/JsonWriterOptions.xml

```diff
@@ -34,7 +37,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Defines whether the <see cref="T:System.Text.Json.Utf8JsonWriter" /> should pretty print the JSON which includes:
```

Comment:
@ahsonkhan pretty print the JSON -> can I replace this with "format the JSON to render nicely" or something to that extent?

Reply:
Sure. Indented/formatted/with white space. Any of that works.